Multisig/Data/Services/Safe Client Gateway Service/SCGModels.swift

```diff
@@ -529,6 +529,28 @@ extension SCGModels {
         var fallbackHandler: AddressInfoExtended?
         var version: String
     }
+
+
+    struct Chain: Decodable {
+        let chainId: String
+        let chainName: String
+        let rpcUrl: String
```

Comment:
URL?

Reply:
Actually in the same file we receive urls as both URL&String, so I kept it as string to avoid handling type mismatching.